src/adhocracy_frontend/adhocracy_frontend/static/i18n/core_de.json

```diff
@@ -33,6 +34,7 @@
     "TR__EDIT_PROFILE_IMAGE": "Profilbild bearbeiten",
     "TR__EMAIL": "E-Mail",
     "TR__EMAIL_OR_USERNAME": "E-Mail oder Benutzername",
+    "TR__ENTER_CAPTCHA": "Captcha eingeben",
```

Comment:
"Captcha lösen"?  "Lösung eingeben"?  If this is the error message displayed when the input field is empty, then "Captcha eingeben" is also ok.